Pipfile

```diff
@@ -10,6 +10,7 @@ ipython = "*"
 
 [packages]
 colorama = ">=0.3.8"
+readchar = "*"
```

Comment:
I was about to merge the PR but I'm actually concerned about the lack of specified version.
Are we certain that any version is fine? Can't we specify at least ">=2.0.0"?